.github/workflows/build_debian.yml

```diff
@@ -0,0 +1,121 @@
+name: Build Debian source package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  check_version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout codebase
+        uses: actions/checkout@v4
+      
+      - name: Extract version from release tag
+        id: version
+        run: 
+          VERSION=${GITHUB_REF#refs/tags/v}
+      
+      - name: Extract version from changelog
+        id: changelog_version
+        run: |
+        CHANGELOG_VERSION=$(dpkg-parsechangelog -S Version | sed -rne 's,([^-\+]+)+(\+dfsg)*.*,\1,p'i)
```

Comment:
I wonder if we should require that the version information include the fully specified version? So something like `0.5.0-0ubuntu2` - otherwise we wouldn't be able to trigger a 'packaging only' release using this workflow.